intermediate-exercises/src/components/01-LifeCycle.js

```diff
@@ -1,44 +1,41 @@
 import React from 'react';
 
-/*
-  Tasks for this exercise are in comments inside the component class code.
-  In this exercise lifecycle methods will be used to provide convenient debug
-  messages in developer's console.
-  There are more lifecycle methods available.
-  Those three presented are commonly used.
- */
-
 class LifeCycle extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { name: 'Pesto' };
+    this.state = { name: 'Good Morning, Pesto!' };
```

Comment:
There is no need to change the variable name. Instructions are to change the `name` state when the button is clicked, not otherwise.